src/components/form/FormComponent.vue

```diff
@@ -1,81 +1,81 @@
 <template>
   <section>
-    <div class="container">
-      <div class="row" v-for="row in rowArray" :key="row.id">
-        <div class="col" v-if="row.isButtonRow">
-          <component
-              :is="checkVueType(field)"
-              v-for="field in row.fields"
-              :key="field.propertyName"
-              :field="field"
-              @onclick="handleButtonClicked" />
-        </div>
-        <template
-            v-for="field in row.fields"
-            v-else>
-            <component
-              :is="checkVueType(field)"
-              v-if="
-                checkVueType(field) ===
-                'FormSection'
-              "
-              :key="field.propertyName"
-              v-model="field.value"
-              :field="field" />
-            <template v-else>
-              <div class="col"
-                v-show="showField(field)"
+    <div class="container" v-if="forms">
+      <div class="row" v-for="formRow in forms" :key="formRow">
+        <div class="col" v-if="formRow.fields">
+          <template v-for="field in formRow.fields">
+              <component
+                :is="checkVueType(field)"
+                v-if="
+                  checkVueType(field) ===
+                  'FormSection'
+                "
                 :key="field.propertyName"
-                :class="
-                  expressCell(field.expression)
-                ">
-                <label
-                  v-if="
-                    !hideLabelForType(
-                      field.vueType
-                    )
-                  "
-                  :for="field.propertyName"
-                  :title="field.title"
-                  :class="{
-                    mandatory: field.mandatory,
-                  }">
-                  {{ field.title }}
-                </label>
-              </div>
-              <div class="col-xl"
-                :key="field.propertyName"
-                :class="
-                  expressCell(field.expression)
-                ">
+                v-model="field.value"
+                :field="field" />
+              <template v-else>
+                <div class="col"
+                  v-show="showField(field)"
+                  :key="field.propertyName"
+                  :class="
+                    expressCell(field.expression)
+                  ">
+                  <label
+                    v-if="
+                      !hideLabelForType(
+                        field.vueType
+                      )
+                    "
+                    :for="field.propertyName"
+                    :title="field.title"
+                    :class="{
+                      mandatory: field.mandatory,
+                    }">
+                    {{ field.title }}
+                  </label>
+                </div>
+                <div class="col-xl"
```

Comment:
Codacy has a fix for the issue: [Expected a linebreak before this attribute.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
              <div
```